rank/data/tests/works.json

```diff
@@ -117,8 +116,21 @@
                 "ratings": [
                     "ruedafcw"
                 ],
-                "description": "Example of a known title's prefix, but not the full thing",
-                "knownFailure": true
+                "description": "Example of a known title's prefix, but not the full thing"
+            },
+            {
+                "searchTerms": "mammas favorites",
+                "ratings": [
+                    "dbqsn5gk"
+                ],
+                "description": "Searching without punctuation should match a document with punctuation"
+            },
+            {
+                "searchTerms": "sophies shell",
+                "ratings": [
+                    "gdfhp4gw"
+                ],
+                "description": "Searching without punctuation should match a document with punctuation"
             }
```

Comment:
Would it be helpful to check that searches _with_ punctuation continue to work, or is that not necessary?

e.g. that searching for "sophie's shell" also finds gdfhp4gw

Reply:
I think it's very unlikely that we'd miss on a search like that, but we've been burned by that kind of assumption before 😅 

will add that test now

Reply:
I'm glad that you asked this question! Using the candidate query/mapping and searching for `sophie's shell` brings back `gdfhp4gw` 3rd in the list of results, while `sophies shell` brings it back 1st 🤔 

I think that's acceptable for now, but obviously not ideal. Another one to puzzle about in the next round of changes I think.